### PR TITLE
Added a solution/fixed stub

### PR DESCRIPTION
**Pull request by jan-miksik**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: guide/english/certifications/coding-interview-prep/rosetta-code/100-doors/index.md [hunk 1]
```diff
@@ -3,8 +3,87 @@ title: 100 doors
...
+
+![:rocket:](https://forum.freecodecamp.com/images/emoji/emoji_one/rocket.png?v=3 ":rocket:") <a href='https://repl.it/CLnD/0' target='_blank' rel='nofollow'>Run Code</a>
+`
+
+## ![:clipboard:](https://forum.freecodecamp.com/images/emoji/emoji_one/clipboard.png?v=3 ":clipboard:") NOTES FOR CONTRIBUTIONS:
+
+*   ![:warning:](https://forum.freecodecamp.com/images/emoji/emoji_one/warning.png?v=3 ":warning:") **DO NOT** add solutions that are similar to any existing solutions. If you think it is **_similar but better_**, then try to merge (or replace) the existing similar solution.
+*   Add an explanation of your solution.
+*   Categorize the solution in one of the following categories — **Basic**, **Intermediate** and **Advanced**. ![:traffic_light:](https://forum.freecodecamp.com/images/emoji/emoji_one/traffic_light.png?v=3 ":traffic_light:")
+*   Please add your username only if you have added any **relevant main contents**. (![:warning:](https://forum.freecodecamp.com/images/emoji/emoji_one/warning.png?v=3 ":warning:") **_DO NOT_** _remove any existing usernames_)
```
Comment (lipis) [inline guide/english/certifications/coding-interview-prep/rosetta-code/100-doors/index.md]:
```suggestion
- Please add your username only if you have added any **relevant main contents**. (![:warning:](https://forum.freecodecamp.com/images/emoji/emoji_one/warning.png?v=3 ":warning:") **_DO NOT_** _remove any existing usernames_)
```

File: guide/english/certifications/coding-interview-prep/rosetta-code/100-doors/index.md [hunk 2]
```diff
@@ -3,8 +3,87 @@ title: 100 doors
...
+
+
+![:rocket:](https://forum.freecodecamp.com/images/emoji/emoji_one/rocket.png?v=3 ":rocket:") <a href='https://repl.it/CLnD/0' target='_blank' rel='nofollow'>Run Code</a>
+`
+
+## ![:clipboard:](https://forum.freecodecamp.com/images/emoji/emoji_one/clipboard.png?v=3 ":clipboard:") NOTES FOR CONTRIBUTIONS:
+
+*   ![:warning:](https://forum.freecodecamp.com/images/emoji/emoji_one/warning.png?v=3 ":warning:") **DO NOT** add solutions that are similar to any existing solutions. If you think it is **_similar but better_**, then try to merge (or replace) the existing similar solution.
+*   Add an explanation of your solution.
+*   Categorize the solution in one of the following categories — **Basic**, **Intermediate** and **Advanced**. ![:traffic_light:](https://forum.freecodecamp.com/images/emoji/emoji_one/traffic_light.png?v=3 ":traffic_light:")
```
Comment (lipis) [inline guide/english/certifications/coding-interview-prep/rosetta-code/100-doors/index.md]:
```suggestion
- Categorize the solution in one of the following categories — **Basic**, **Intermediate** and **Advanced**. ![:traffic_light:](https://forum.freecodecamp.com/images/emoji/emoji_one/traffic_light.png?v=3 ":traffic_light:")
```

File: guide/english/certifications/coding-interview-prep/rosetta-code/100-doors/index.md [hunk 3]
```diff
@@ -3,8 +3,87 @@ title: 100 doors
...
+    console.log(JSON.stringify(getFinalOpenedDoors(100))); // return [1,4,9,16,25,36,49,64,81,100]
+
+
+![:rocket:](https://forum.freecodecamp.com/images/emoji/emoji_one/rocket.png?v=3 ":rocket:") <a href='https://repl.it/CLnD/0' target='_blank' rel='nofollow'>Run Code</a>
+`
+
+## ![:clipboard:](https://forum.freecodecamp.com/images/emoji/emoji_one/clipboard.png?v=3 ":clipboard:") NOTES FOR CONTRIBUTIONS:
+
+*   ![:warning:](https://forum.freecodecamp.com/images/emoji/emoji_one/warning.png?v=3 ":warning:") **DO NOT** add solutions that are similar to any existing solutions. If you think it is **_similar but better_**, then try to merge (or replace) the existing similar solution.
+*   Add an explanation of your solution.
```
Comment (lipis) [inline guide/english/certifications/coding-interview-prep/rosetta-code/100-doors/index.md]:
```suggestion
- Add an explanation of your solution.
```

File: guide/english/certifications/coding-interview-prep/rosetta-code/100-doors/index.md [hunk 4]
```diff
@@ -3,8 +3,87 @@ title: 100 doors
...
+
+    console.log(JSON.stringify(getFinalOpenedDoors(100))); // return [1,4,9,16,25,36,49,64,81,100]
+
+
+![:rocket:](https://forum.freecodecamp.com/images/emoji/emoji_one/rocket.png?v=3 ":rocket:") <a href='https://repl.it/CLnD/0' target='_blank' rel='nofollow'>Run Code</a>
+`
+
+## ![:clipboard:](https://forum.freecodecamp.com/images/emoji/emoji_one/clipboard.png?v=3 ":clipboard:") NOTES FOR CONTRIBUTIONS:
+
+*   ![:warning:](https://forum.freecodecamp.com/images/emoji/emoji_one/warning.png?v=3 ":warning:") **DO NOT** add solutions that are similar to any existing solutions. If you think it is **_similar but better_**, then try to merge (or replace) the existing similar solution.
```
Comment (lipis) [inline guide/english/certifications/coding-interview-prep/rosetta-code/100-doors/index.md]:
```suggestion
- ![:warning:](https://forum.freecodecamp.com/images/emoji/emoji_one/warning.png?v=3 ":warning:") **DO NOT** add solutions that are similar to any existing solutions. If you think it is **_similar but better_**, then try to merge (or replace) the existing similar solution.
```

File: guide/english/certifications/coding-interview-prep/rosetta-code/100-doors/index.md [hunk 5]
```diff
@@ -3,8 +3,87 @@ title: 100 doors
...
+
+## Spoiler Alert!
+
+![warning sign](//discourse-user-assets.s3.amazonaws.com/original/2X/2/2d6c412a50797771301e7ceabd554cef4edcd74d.gif)
+
+**Solution ahead!**
+
+## ![:beginner:](https://forum.freecodecamp.com/images/emoji/emoji_one/beginner.png?v=3 ":beginner:") Basic Code Solution:
+
+    function getFinalOpenedDoors (numDoors) {
```
Comment (thecodingaviator) [inline guide/english/certifications/coding-interview-prep/rosetta-code/100-doors/index.md]: Put this in code block with \`\`\`js

**Review by thecodingaviator — CHANGES_REQUESTED** [review]

**Comment by RandellDawson** [comment]
@lipis @thecodingaviator This PR has gone stale.  I suggest either making the changes on behalf of the contributor or close this PR.

**Review by thecodingaviator — COMMENTED** [review]
Hi,

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened, if you still wish to make the changes you originally added.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding! ✈️

**Pull request closed without merging**